### DISCUSSION TITLE
Cardano support

### DISCUSSION THREAD

**Discussion by santonode** [snapshot-labs/snapshot#3079] [General]
Can Snapshot support Cardano?

**Comment by metrox-eth** [comment]
Snapshot is open source and work natively with EVM chains, if you want to make Snapshot works with a non-EVM chain like Cardano you may want to find resources to do it as a community effort. It will require some developement.

**Reply by santonode** [reply]
Cardano supports EVM sidechains
https://iohk.io/en/blog/posts/2022/07/06/introducing-the-cardano-evm-sidechain/

**Reply by metrox-eth** [reply]
AFAIK a side chain is not the mainnet. Cardano is still non-EVM.

**Reply by HarmonicPool** [reply]
@metrox-eth can you point me to the lines that take care of fetching the data? It's not clear to me how the application layer influences the calculation of number of token holders

**Reply by metrox-eth** [reply]
Snapshot use about 350 different javascript strategies, the strategies read balances of ERC20, 721, 1155 and define how the voting power is calculated from there, then the voting systems takes those values and calculate the results of a proposal. You also need to authenticate an user and his actions on Cardano, the easiest way to integrate is to make users on Cardano delegate to an Ethereum address and use that Eth address to vote. You can reach to @bonustrack for more details.

**Comment by santonode** [comment]
Is there somebody else I can talk to? We have over 100 DAOs on Cardano already including my own freeloaderz.io I can bring them to snapshot because we need governance tooling.

Developer tool compatibility. Ethereum developer tools are widely used for asset storage, reading the state of the ledger, monitoring analytics, and more. EVM sidechain users will have access to these tools from the Cardano ecosystem. This creates a lower barrier to entry for Solidity developers looking to build on Cardano.

Web3.js wallet compatibility. Web3 technology provides users with complete control over their own data. This means no involvement of third-party corporations owning users’ personal data. Web3 wallets are known for ensuring user anonymity and data protection while interacting with DApps.

**Reply by metrox-eth** [reply]
Awesome, I am not saying it's not possible, I just say it will require development, if there is 100 DAO it should be possible to find a few dev who want to dig into this.
